src/csharp/Grpc.Core.Tests/Grpc.Core.Tests.csproj

```diff
@@ -8,7 +8,7 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
 
-  <PropertyGroup Condition=" '$(TargetFramework)' == 'netcoreapp2.1' ">
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'netcoreapp2.1' or '$(TargetFramework)' == 'net45' ">
```

Comment:
btw, the test project uses `<TargetFrameworks>net45;netcoreapp2.1</TargetFrameworks>` so by making this change, we are no longer testing the codepath in Grpc.Core with GRPC_CSHARP_SUPPORT_SYSTEM_MEMORY=false.
I might need to add an extra testing project to make sure we visit that codepath.